packages/mermaid/src/mermaid.ts

```diff
@@ -247,9 +230,6 @@ const mermaid: {
   initialize: typeof initialize;
   contentLoaded: typeof contentLoaded;
   setParseErrorHandler: typeof setParseErrorHandler;
-  // Array of functions to use for detecting diagram types
-  detectors: Array<any>; // eslint-disable-line @typescript-eslint/no-explicit-any
-  connectDiagram: (id: string, diagram: DiagramDefinition, callback: (id: string) => void) => void;
```

Comment:
@knsv I've removed detectors and connect diagram for now as we're currently using lazyLoadedDiagrams.
We can introduce it later once we have nailed down the interface completely.